apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtaksvurderingRepository.kt

```diff
@@ -326,7 +327,21 @@ class VedtaksvurderingRepository(private val datasource: DataSource) : Transacti
                     ),
                 loggtekst = "Attesterer vedtak $behandlingId",
             )
-                .also { require(it == 1) }
+                .also {
+                    require(it == 1)
+                }
+
+            opprett(
+                query = """
+                    INSERT INTO outbox_vedtakshendelse (vedtakId, type) 
+                    SELECT v.id, 'ATTESTERT'
+                    FROM vedtak v
+                    WHERE v.behandlingId = :behandlingId
+                    """,
+                params = mapOf("behandlingId" to behandlingId),
+                loggtekst = "Lagt til innslag for attestert vedtak i outbox",
```

Comment:
Ser at loggingen her får `javax.sql.DataSource` som `component`. Antar at dette ble innført for å spare noen loggere histen og pisten men jeg synes det ser rart ut.

https://logs.adeo.no/app/r/s/wN5XG.